stdlib/grpc/src/main/resources/templates/skeleton/service_sample.mustache

```diff
@@ -38,3 +38,12 @@ service {{serviceName}} on ep {
         // You should return a {{outputType}}
     }{{/unaryFunctions}}
 }
+
+{{#each messageList}}{{> message}}{{/each}}
+{{#enumList}}{{> enum}}{{/enumList}}{{#if rootDescriptor}}const string ROOT_DESCRIPTOR = "{{rootDescriptor}}";
```

Comment:
```suggestion
{{#enumList}}{{> enum}}{{/enumList}}{{#if rootDescriptor}}
const string ROOT_DESCRIPTOR = "{{rootDescriptor}}";
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/21024/commits/f18fc70da127f866e12b0df7937719b6a4259a1b